src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -0,0 +1,32 @@
+import { ICP_NETWORK } from '$env/networks.env';
+import { BTC_MAINNET_TOKEN } from '$env/tokens.btc.env';
+import { ETHEREUM_TOKEN, ICP_TOKEN } from '$env/tokens.env';
+import type { Token } from '$lib/types/token';
+import { nonNullish } from '@dfinity/utils';
+
+const PINNED_TOKENS_ORDER = [
+	ICP_TOKEN,
+	BTC_MAINNET_TOKEN,
+	ETHEREUM_TOKEN,
+	{ id: Symbol('ckBTC'), network: ICP_NETWORK },
```

Comment:
This feels like it could be "dangerous". We are using symbols because we want to be absolutely sure we are referencing the same unique token across the entire application. By creating a particular symbol here, we are creating a new reference. This is a pattern that is prone to leading to errors, so I'm not sure what the exact goal is?

Reply:
I agree, I was trying to get the list of ckBTC, ckETH and ckUSDC here, but we don't really have it somewhere... shall i create it as Twin Tokens?

Reply:
Ideally we should not create new information. I think we probably have already enough data. There is probably a way. For example I would use the existing `ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS`.

Reply:
done, have a look